rpcserver.go

```diff
@@ -2434,7 +2559,8 @@ func SendToAddress(w *Wallet, chainSvr *chain.Client, icmd btcjson.Cmd) (interfa
 		cmd.Address: btcutil.Amount(cmd.Amount),
 	}
 
-	return sendPairs(w, chainSvr, cmd, pairs, 1)
+	// sendtoaddress always debits the default account, this matches bitcoind
```

Comment:
This isn't incorrect, but it behaves differently.  We group utxos by account, but bitcoind does not.  Debiting from an account may spend *any* available input, and simply decrements the balance associated with that account.

That said, I'm ok with sendtoaddress only spending from the default account's utxos, as it's the least surprising behavior, and other RPCs exist for spending out of other accounts.